app/Blogs/[slug]/page.js

```diff
@@ -0,0 +1,17 @@
+"use client";
+import { usePathname, useSearchParams } from "next/navigation";
+
+export default function BlogPost() {
+  const path = usePathname();
+  const slug = path.split("/")[2];
```

Comment:
Instead of using usePathName here, you could have used the [params prop](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes). Also, instead of getting the third item in this array with [2] would it not make more sense to get the last item?

Reply:
I worked with using param props, that was more simple thanks for suggestion. I hope I did it correctly